pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py

```diff
@@ -74,7 +74,8 @@ def upgrade():
     conn = op.get_bind()
     inspector = Inspector.from_engine(conn)
     tables = inspector.get_table_names()
-    metadata = MetaData(conn, reflect=True)
+    metadata = MetaData(conn)
+    metadata.reflect()
```

Comment:
SqlAlchemy fix when we upgrade. I tried upgrading to 1.4, but fell back after I realized it wasn't necessary to upgrade.